config/jobs/kubernetes/sig-gcp/gpu/sig-gcp-gpu-gce.yaml

```diff
@@ -2,10 +2,8 @@ presets:
 - labels:
     preset-ci-gce-device-plugin-gpu: "true"
   env:
-  - name: KUBE_FEATURE_GATES
-    value: DevicePlugins=true
   - name: KUBE_GCE_NODE_IMAGE
-    value: gke-1-8-4-gke-1-cos-stable-63-10032-71-0-p-v171208-pre-nvda-gpu
+    value: gke-1134-gke-rc5-cos-69-10895-138-0-v190320-pre-nvda-gpu
```

Comment:
FYI @vpickard @SergeyKanzhelev
I just stumbled across this while working on https://github.com/kubernetes/k8s.io/issues/1095

I don't know if this is at risk of falling out of date at some point

Reply:
@jiayingz Can you provide some reference on the lifespan on this image? Any links would be very helpful. We have been addressing issues in sig-node e2e tests related to outdated images. Also, I'm wondering what triggered the update of this image? Was it to enable additional functionality with a newer image?